src/count.cpp

```diff
@@ -0,0 +1,21 @@
+/*
+OVERVIEW:  Given 2 integers (num1, num2), write a function that returns the count of numbers between the num1..num2 that are divisible by num1.
+		E.g.: count(3, 15) returns 5 (3, 6, 9, 12, 15).
+		Note : You need to consider Inclusive range
+INPUTS:  Two numbers num1, num2.
+		num1, num2 >= 0.
+
+OUTPUT: Return the count of numbers between the num1..num2 that are divisible by num1.
+
+ERROR CASES: Return -1 in error cases.
+
+NOTES:
+*/
+int count(int num1, int num2) {
+	if (num1 == 0)return -1;
+	if (num2 == 0)return 0;
+	if (num1 > num2)return 1;
+	int x = 0;
+	if (num2%num1 == 0)x = 1;
+	return (num2 - num1 - 1) /num1 + 1+x;
```

Comment:
Good to see that you've arrived with some mathematical solution which is of O(1) complexity. But, I guess you're a bit confused about its logic. Re-think about it once, get all the thoughts together, I feel that lines `18-20` can be written in a single line without even making that many operations. A single operation with a single line would be sufficient. Think about it and update the function and commit.

Reply:
Anna for the question countFactorial I used 17 as array size because after 16! it's value exceed size of an integer. So, to avoid that I used 17.
If in case we are dealing with highest numbers then we can use long type and store highest values.